site/docs/mfa/webhooks/about.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: aboutMfaWebhooks
+title: MFA Webhooks
+slug: /mfa/webhooks
+description: About Bandwidth's MFA Webhooks
+keywords:
+  - mfa
+  - callbacks
+  - webhooks
+  - bandwidth
+hide_title: false
+sidebar_label: About
+custom_edit_url: github.com/Bandwidth/api-docs/tree/main/site/docs/mfa/webhooks/about.mdx
+---
+# Two-Factor Authentication Webhooks
+Two-Factor Authentication (2FA) Webhooks (also known as Callbacks) rely on DLRs (Delivery Receipts) sent to the Two-Factor Authentication (2FA) platform by both Bandwidth's North America and International platform. Since these two platforms send different DLR formats, the 2FA platform standardizes them into a singular format for simplicity and ease of consumption. The 2FA platform then takes these standardized DLRs and sends them via an HTTP POST to the specified URL that was set in the `callbackUrl` field in the [initial request](../../messaging/quickStart.mdx#configuring-callback-urls). The `callbackUrl` field can be any publicly addressable url.
+
+## Message Webhooks Concepts
+- Both North America and International webhooks have the same fields and have the same format. However, some of these fields differ in values to denote between the two systems.
```

Comment:
Suggestion:
Both North America and International webhooks have the same fields and format. However, some of these fields have different values between the two systems.

It would be nice if we could include a couple examples in parenthesis or a follow-on sentence

Reply:
updated